pootle/apps/pootle_app/management/commands/contributors.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import os
+os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
+
+from collections import Counter
+from optparse import make_option
+from django.contrib.auth import get_user_model
+from pootle_app.management.commands import PootleCommand
+from pootle_store.models import Unit
```

Comment:
We already discussed it. To refresh your memory: Unnecessary newlines and special casing for no good reason of django and ttk.